scripts/service/timeless_pg.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=PostgreSQL DB Timeless
+Documentation=man:postgres(1)
+
+[Service]
+Type=forking
+User=postgres
+#pg_conftool show -s data_directory;
```

Comment:
@marceloamadeu Is it comment line? Why do we need it?

Reply:
@vladarefiev Commands included in install_db.